Apps/AdminWebClient/src/Server/Services/CommunicationService.cs

```diff
@@ -64,7 +64,7 @@ public RequestResult<Communication> Add(Communication communication)
         /// <inheritdoc />
         public RequestResult<Communication> Update(Communication communication)
         {
-            if (ValidateDates(communication.EffectiveDateTime, communication.ExpiryDateTime))
+            if (communication.CommunicationTypeCode == CommunicationType.Email || ValidateDates(communication.EffectiveDateTime, communication.ExpiryDateTime))
```

Comment:
I thik @sam-warren is working on the service, coordinate with him to make sure this doesn't cause unnecessary conflicts

Reply:
It should be ok I think, @sam-warren can you please confirm?